### ISSUE TITLE
The extension doesn't correctly handle location intervals spanning several lines

### ISSUE THREAD

**Issue by lukaszcz** [anoma/vscode-juvix#60]
This is the reason for #24. It also makes errors spanning several lines highlight incorrectly.

**Comment by jonaprieto** [comment]
Could you please provide a screenshot for me to examine? I have a suspicion about what might be happening.

**Comment by lukaszcz** [comment]
Actually, nothing is happening -- just nothing is underlined when the interval spans several lines, when the error message is e.g.:
```
/Users/heliax/Documents/juvix/tests/Compilation/negative/test002.juvix:8-9:5-25: error:
Pattern matching not exhaustive. Example pattern not matched: _ :: nil

 *  The terminal process "/bin/zsh '-c', 'juvix --internal-build-dir .juvix-build  typecheck /Users/heliax/Documents/juvix/tests/Compilation/negative/test002.juvix'" terminated with exit code: 1. 
```
You can see this on the Compilation/negative/test002.juvix test in our tests.

**Comment by vrom911** [comment]
This should be resolved now in #63 
Let me know if it is still there now
